packages/itwinui-css/src/bridge.scss

```diff
@@ -250,14 +250,9 @@
 
   .iui-input:not([role='combobox']) {
     // iTwinUI v5 input background colors are different from global values
```

Comment:
As we're now using `@stratakit` packages, can we also update all comments to use "StrataKit" instead of "iTwinUI v5"?